### ISSUE TITLE
Win auth Confirmation doesnt work.. PLEASE HELP ME

### ISSUE THREAD

**Issue by Addikt993** [winauth/winauth#470]
Thing is on my work place i got steam auth too and there is older version like 3.4 something and at home i accidentaly upgreaded to newest and it doesnt work and when i delete this one and put older one it doesnt work .. confimation is gray and canot be clicked.. PLEASE HELP ME GUYS...

**Comment by nyakze** [comment]
Steam must have changed something, doesn't work for me as well.

**Comment by quickfiya** [comment]
Yeah, it won't work for me either. I've tried sync time and everything

**Comment by nyakze** [comment]
Managed to get it to work on SDA. Just had to relogin.

**Comment by Dhaya06** [comment]
Winauth not working. I tried with even my recovery code. It keep saying wrong verification. I have activated 2 step verification to my uplay account. Now i cant access anything. Please help me. What should i do.

**Comment by nyakze** [comment]
I haven't used authenticator for uplay, but the common course of action is to contact uplay support, tell them you lost your authenticator and ask them to remove it.
